docs/plugins/zero.rst

```diff
@@ -2,8 +2,10 @@ Zero Plugin
 ===========
 
 The ``zero`` plugin allows you to null fields in files' metadata tags. Fields
-can be nulled unconditionally or conditioned on a pattern match. For example,
-the plugin can strip useless comments like "ripped by MyGreatRipper."
+can be nulled unconditionally or conditioned on a pattern match. It works in
+two independent modes - blacklist and whitelist. You can only choose one option,
+however blacklist is the default. For example, the plugin can strip useless
+comments like "ripped by MyGreatRipper."
```

Comment:
One more writing suggestion: instead of adding to this paragraph, add a new one below (after the example).

> The plugin can work in one of two modes. The first mode, the default, is a *blacklist*, where you choose the tags you want to remove. The second mode is a *whitelist* where you instead specify the tags you want to keep.